packages/core/src/external/carequality/iti-55-parsing.ts

```diff
@@ -21,6 +23,10 @@ async function parseXmlString(xml: string): Promise<[PatientData, [string, strin
   try {
     //eslint-disable-next-line @typescript-eslint/no-explicit-any
     const result = await parser.parseStringPromise(xml);
+    const queryId =
+      result["Envelope"]["Body"][0]["PRPA_IN201305UV02"][0]["controlActProcess"][0][
+        "queryByParameter"
+      ][0]["queryId"][0]["$"]["extension"];
```

Comment:
What if one of these is not present?

The compiler should be complaining about this, as we have `noUncheckedIndexedAccess` set on `tsconfig`. You don't get an error when trying to compile it?

Reply:
I am not sure why my compiler isnt complaining. I tried to debug for a while. Will make those changes but maybe we can investigate my env later?